cli/src/cargo_circus.rs

```diff
@@ -1,98 +1,67 @@
 mod common;
 
+use circus_cli_options::{Command, Options};
 use clap::Parser;
-use common::options::{circus_engine_part, wrapped};
-use std::process::Command;
 
-const ENGINE_BINARY_NAME: &str = "circus-engine";
-const ENGINE_BINARY_NOT_FOUND: &str = const_format::formatcp!(
-    "The binary [{}] was not found in your [PATH].",
-    ENGINE_BINARY_NAME,
-);
+fn run(options: Options) -> i32 {
+    let exit_status = std::process::Command::new("cargo")
+        .args(["build".into()].iter().chain(options.cargo_flags.iter()))
```

Comment:
Do you want to add the `+nightly-...` things later?

Reply:
Oh, maybe I can do a quick something already yes

Reply:
done in b433339ac576a4b89409aa38dea0d701476fcf23